test/functional-tests/pageObjects/ccd-components/fields/ccdDateField.js

```diff
@@ -41,18 +41,41 @@ class CcdDateField{
     await this._enterDateWithParams(day, month, year);
   }
 
+   /**
+   * Check if field is ready to type
+   * @returns true or false
+   */
+  async isFieldInputReady(){
```

Comment:
small thing but could possibly change this to  areFieldInputsReady to make it clear we are waiting for multiple fields here